core/audits/network-requests.js

```diff
@@ -77,6 +88,8 @@ class NetworkRequests extends Audit {
         priority: record.priority,
         isLinkPreload,
         experimentalFromMainFrame,
+        isFirstParty,
+        isThirdParty,
```

Comment:
one thing we had talked about was adding `entityName` to make look up into `lhr.entities` easier. Is third/first party differentiation all you're looking for?

Reply:
+ 1 to `entityName` over `isFirstParty`/`isThirdParty`

Reply:
@brendankenny tbh I was still noodling around with it and threw up the draft PR as a placeholder to finalise. sorry I didn't call that out in the description! I agree, entity names is the way to go